src/uthash.h

```diff
@@ -423,7 +480,7 @@ do {
  */
 #ifdef HASH_DEBUG
 #define HASH_OOPS(...) do { fprintf(stderr,__VA_ARGS__); exit(-1); } while (0)
-#define HASH_FSCK(hh,head)                                                       \
+#define HASH_FSCK(hh,head,fun)                                                   \
```

Comment:
Oh, and parenthesize `(where)` every time it's used in this macro, for hygiene. Doesn't matter for correctness, but I'd like to keep a general rule of "hygienic macros all the time no matter what."